website/www/site/content/en/contribute/_index.md

```diff
@@ -140,122 +193,109 @@ You can install these in a Debian-based distribution for Linux or macOs using th
 * Python
 * Docker
 
-To install:
-
-1. Execute:
+To install execute:
 ```
 ./local-env-setup.sh
 ```
 
-#### Container: Docker-based
+##### Container: Docker-based
 
 Alternatively, you can use the Docker based local development environment to wrap your clone of the Beam repo
 into a container meeting the requirements above.
 
-You can start this container using the [start-build-env.sh](https://github.com/apache/beam/blob/master/start-build-env.sh)
-script which is part of the Beam repo:
+You can start this container using the [start-build-env.sh](https://github.com/apache/beam/blob/master/start-build-env.sh) script which is part of the Beam repo.
 
-1. Execute:
+Execute:
 ```
 ./start-build-env.sh
 ```
 
-### Connect With the Beam community
+#### Development Setup {#development-setup}
 
-1. Consider subscribing to the [dev@ mailing list](/community/contact-us/), especially
-   if you plan to make more than one change or the change will be large. All decisions happen on the
-   public dev list.
-1. (Optionally) Join the [#beam channel of the ASF slack](/community/contact-us/).
-1. Create an account on [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues)
-   (anyone can do this).
+1. Check [Git workflow tips](https://cwiki.apache.org/confluence/display/BEAM/Git+Tips) if you need help with git forking, cloning, branching, committing, pull requests, and squashing commits.
 
-### Share your intent
+2. Clone the git repository. You can download it anywhere you like.
+```
+$ mkdir -p ~/go/src/github.com/apache
```

Comment:
Is this to ensure that the Beam is on the Go path? Otherwise I don't understand why you'd be creating a new directory path under `~/go`... I've always followed these steps: https://cwiki.apache.org/confluence/display/BEAM/Git+Tips

Reply:
Alternatively, if we want to state that people need to download the repo right away, we can change the code to: 
```
$ mkdir -p ~/path/to/your/folder
$ cd ~/path/to/your/folder
$ git clone https://github.com/apache/beam
$ cd beam
```

Please LMK which option works better in your opinion.

Reply:
> It's better to use these command (like it is written now).

Why? Sorry if I'm missing something... I would think that you want to 1) Fork the repo; 2) clone the repo to your workspace. Cloning the repo onto your Go path is only necessary for Go development, right? It seems like we should call that out. Also, wouldn't you want to fork the repo before cloning it, no matter where you clone it? Again, please let me know if I'm missing something...

Reply:
You are right. Let's rewrite this one to:
```
1. Check [Git workflow tips](https://cwiki.apache.org/confluence/display/BEAM/Git+Tips) if you need help with git forking, cloning, branching, committing, pull requests, and squashing commits.
2. Make a fork
3. Clone the forked repository. You can download it anywhere you like.

$ mkdir -p ~/path/to/your/folder
$ cd ~/path/to/your/folder
$ git clone https://github.com/forked/apache/beam
$ cd beam

```
What do you think about this one?

Reply:
LGTM

Reply:
changed